src/commands/mod.rs

```diff
@@ -747,12 +751,15 @@ impl DaemonControl {
         let timelock =
             timelock.unwrap_or_else(|| self.config.main_descriptor.first_timelock_value());
         let height_delta: i32 = timelock.try_into().expect("Must fit, it's a u16");
-        let sweepable_coins = db_conn.coins(CoinType::Unspent).into_values().filter(|c| {
-            // We are interested in coins available at the *next* block
-            c.block_info
-                .map(|b| current_height + 1 >= b.height + height_delta)
-                .unwrap_or(false)
-        });
+        let sweepable_coins = db_conn
+            .coins(Some(&[CoinStatus::Confirmed]), None)
```

Comment:
That's a behaviour change. Previously we'd query all unspent coins, including unconfirmed ones. I think just including `CoinStatus::Confirmed` too should fix this.

Reply:
Ah yes, I'll include both confirmed and unconfirmed coins here.

Reply:
I've now updated this.